description: remove strict clang version check during config
author: Andres Salomon <dilinger@debian.org>

Chromium 107 has a strict clang version check, added in commit
8f23a2c2d14fd799813134e995c160354d75d3a0. This needs a proper fix
upstream; some way to check (or specify) whether it's a distribution
build, and therefore shouldn't require a particular git version of
clang.

For now, let's just get this building in debian.

--- a/tools/clang/scripts/update.py
+++ b/tools/clang/scripts/update.py
@@ -355,7 +355,7 @@ def main():
       return 0
 
     stamp_version = ReadStampFile(STAMP_FILE).partition(',')[0]
-    if PACKAGE_VERSION != stamp_version:
+    if False:
       print('The expected clang version is %s but the actual version is %s' %
             (PACKAGE_VERSION, stamp_version))
       print('Did you run "gclient sync"?')
